crypto/engine/engine.go

```diff
@@ -0,0 +1,145 @@
+/*
+ * Nuts crypto
+ * Copyright (C) 2019. Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package engine
+
+import (
+	"crypto"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/nuts-foundation/nuts-node/core"
+	crypto2 "github.com/nuts-foundation/nuts-node/crypto"
```

Comment:
why2?

Reply:
default golib `crypto` package is also imported